src/main/java/org/wildfly/security/password/util/ModularCrypt.java

```diff
@@ -70,13 +59,7 @@ private PasswordUtil() {}
     private static final int A_APACHE_HTDIGEST          = 7;
     private static final int A_BSD_CRYPT_DES            = 8;
     private static final int A_CRYPT_DES                = 9;
-    private static final int A_DIGEST_MD2               = 10;
```

Comment:
That is what I did in https://github.com/wildfly-security/wildfly-elytron/pull/238.

Is that it ?